products/digital-sales-rooms/app-deployment/cloudflare.md

```diff
@@ -0,0 +1,41 @@
+---
+nav:
+   title: Cloudflare
+   position: 10
+
+---
+
+# Deploy with Cloudflare
+
+In this chapter you will learn how to
+
+- Deploy the template to Cloudflare Pages
+
+## Setup
+- Because of this [issue](https://github.com/nuxt/nuxt/issues/28248), just make sure your `.npmrc` file have
+```
+shamefully-hoist=true
+strict-peer-dependencies=false
+```
+
+- Install Wrangler
+
+```bash
+pnpm install wrangler --save-dev
+```
+
+- Make sure the Front-end App already [generated .env file](../local-installation/app-installation.md#generate-env-file)
+
+- Build your project for Cloudflare Pages:
+
+```bash
+npx nuxi build --preset=cloudflare_pages
+```
+
+- Deploy, it will ask you to create a project for the first time:
+
+```bash
+wrangler pages deploy dist/
+```
+
+- After finish this, your website is ready and Cloudflare also give you a Front-end App domain. Please use the current domain to config [sales channel domain](../configuration/domain-config.md).
```

Comment:
has this point been removed?

Reply:
because the customer requests more document about Cloudflare, so I had to make some updates for these 2 files. Could you please take a look again, sorry for the inconvenience.
https://github.com/shopware/docs/pull/1450/files#diff-ec540b685d6574a3c15f7f3be275ac92685fd128143d4775f48289613ad689af
https://github.com/shopware/docs/pull/1450/files#diff-b7787d19acc5ba7e29c23419d9fcd58f00d80058a901bb21b131378084bb0fbe